.gitignore

```diff
@@ -0,0 +1,8 @@
+/bin/
```

Comment:
Can you please not check in `.gitignore`? They become masses of cruft with everyone checking in their own little personal ignore sets and effectively claiming directories and files

I think you may need to force push *gasp* to get rid of it completely

Reply:
yeah, it's idiosyncratic, but IMO it's much better for everyone to maintain their own local ignore files

Reply:
Hmm, I saw you force pushed, but the gitignore is still there

Reply:
IIUC it needs to be completely blasted from every commit in the PR to not cause issues in the future

Reply:
Force push was rebasing on develop. I actually forgot to remove the gitignore from the subsequent commit, which is why it's still around.

Reply:
the `.gitignore` file should no longer be an issue.

Reply:
:+1: